libraries/AP_HAL_ChibiOS/hwdef/skyviper-f412-rev1/hwdef.dat

```diff
@@ -6,6 +6,9 @@ MCU STM32F4xx STM32F412Rx
 # board ID for firmware load
 APJ_BOARD_ID 9
 
+# short board name override (23 chars)
```

Comment:
```suggestion
# short board name override (13 chars) to match original max length, cant change because name it used for transmitter bind CRC.
```

Reply:
Added this comment to both skyviper hwdefs and shortened their names to 13 vs 14 chars (which was the intent, correct?).